.github/workflows/release.yml

```diff
@@ -0,0 +1,63 @@
+name: Build and Release
+
+on:
+  push:
+    tags:
+      - '*.*'
+    paths-ignore:
+    - 'docs/**'
+    - '.github/site.yml'
+
+jobs:
+  release-on-push:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ 'openjdk11' ]
+      fail-fast: false
+    steps:
+    - name: Setup Graphviz environment
```

Comment:
what is graphviz used for?

Reply:
javadocs (umlgraph)

Reply:
ah, forgot we're uploading a javadocs jar as part of this release process